### PR TITLE
[docker] Add example for adding more ES nodes within docker-compose (#53)

### PR DESCRIPTION
**Pull request by mschreib28**

### CODE REVIEW

**Review by nor3th — CHANGES_REQUESTED** [review]
Hey @mschreib28 

Thank you very much for this addition. Please move the comments to the README.md file, if needed create an own section/chapter for the cluster approach.

Regards

**Comment by mschreib28** [comment]
@nor3th - updated.

File: docker-compose-multiple-es-nodes.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    container_name: elasticsearch3
+    environment:
+      - cluster.name=docker-cluster
+      - "discovery.zen.ping.unicast.hosts=elasticsearch"
+    ulimits:
+      memlock:
+        soft: -1
+        hard: -1
+    volumes:
+      - esdata2:/usr/share/elasticsearch/data
```
Comment (nor3th) [inline docker-compose-multiple-es-nodes.yml:27]:
```suggestion
      - esdata3:/usr/share/elasticsearch/data
```

**Review by nor3th — CHANGES_REQUESTED** [review]
Thanks for the adaptions. Just one typo to fix ;)

File: docker-compose.yml [hunk 2]
```diff
@@ -11,6 +11,8 @@ services:
       - esdata:/usr/share/elasticsearch/data
     environment:
       - discovery.type=single-node
```
Comment (nor3th) [inline docker-compose.yml:13]: My guess is, that once you are in a cluster setup, you have to also remove this line?

**Comment by SamuelHassine** [comment]
@mschreib28 Can you please take the new reviews in consideration so we can merge your PR? Thanks for your work!

File: docker-compose.yml (hunk 2 above)
Comment (mschreib28) [reply docker-compose.yml:13]:
```suggestion
      # Comment out the line below for single-node
      - discovery.type=single-node
```

File: docker-compose.yml [hunk 3]
```diff
@@ -11,6 +11,8 @@ services:
...
+      # Use the below for a cluster of multiple nodes, and comment out the line above (single-node)
```
Comment (mschreib28) [inline docker-compose.yml:14]:
```suggestion
      # Uncomment line below below for a cluster of multiple nodes
```

**Review by mschreib28 — COMMENTED** [review]
Added changes @nor3th recommended, and put them into suggest commits.

**Comment by mschreib28** [comment]
@SamuelHassine - Changed resolved. Thanks!

**Comment by nor3th** [comment]
@mschreib28 Thank you very much! Could you please solve the conflicts and we should be good to go

**Review by nor3th — APPROVED** [review]

**Comment by mschreib28** [comment]
Fixed merge conflict

**Pull request merged**